dlc-test/src/test/scala/org/bitcoins/dlc/DLCClientIntegrationTest.scala

```diff
@@ -203,9 +197,18 @@ class DLCClientIntegrationTest extends BitcoindRpcTest {
       val localVout = localFundingUtxos.head.outPoint.vout
       val remoteVout = remoteFundingUtxos.head.outPoint.vout
 
+      val outcomes = DLCTestUtil.genOutcomes(numOutcomes)
+      val outcomeHashes =
+        outcomes.map(msg => CryptoUtil.sha256(ByteVector(msg.getBytes)).flip)
+
+      val outcomeMap =
+        outcomeHashes.zip(DLCTestUtil.genValues(numOutcomes, totalInput)).toMap
+      val otherOutcomeMap = outcomeMap.map {
+        case (hash, amt) => (hash, (totalInput - amt).satoshis)
```

Comment:
It'd be nice this if this was it's own generator so we could just do `DLCTestUtil.genContractInfos`

Reply:
Also will be extra nice for when we *eventually* have multiparty dlcs.